pom.xml

```diff
@@ -98,6 +98,13 @@
       <artifactId>metrics</artifactId>
       <version>3.0.0</version>
     </dependency>
+    <!-- test dependencies -->
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore</artifactId>
+      <version>4.4.1</version>
+      <scope>test</scope>
```

Comment:
Do you rather mean to `<exclude>` an old version of this dep from some “nearer” dependency chain? CC @andresrc